src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -110,15 +110,30 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler {
 
   std::string DebugString() const;
 
+  /// Update the total resources of the specified node.
+  ///
+  /// \param node_id Id of a node.
+  /// \param changed_resources Changed resources of a node.
+  void UpdateTotalResources(
```

Comment:
Actually, this method updates both total and available resources inside SchedulingResources. So it is better to rename it to `UpdateResourceCapacity`.